.nvmrc

```diff
@@ -0,0 +1 @@
+v14.19.3
```

Comment:
Is `nvm` the thing most people use? I've been using `nodenv`, as it's essentially the same as `rbenv` but I don't know what the majority of people at dxw are using!

Reply:
That's a good point - I used `nodenv` too - didn't clock the filename!

Reply:
Swapped to using `.node-version` instead - `nvm` looks to be heading for support for this anyway.